### ISSUE TITLE
How about a female keynote presenter

### ISSUE THREAD

**Issue by trippetoe** [Esri/devsummit-feedback#20]
I have been blessed to attend 8 dev summits.  In none of those dev summits have i seen a female keynote speaker or a female on the stage at the closing lunch.  There are clearly many women leaders in our field; it seems that Esri should recognize that and draw from that talent pool when selecting presenters.

**Comment by jgravois** [comment]
i think that's a great idea 👩‍🚀.

**Comment by JimBarry** [comment]
Agreed. I mean, while it's never been our intent to actively exclude females in those two sessions, we need to keep working at inclusiveness and diversity. This has been a very common point of feedback, particularly this year, so we will work harder at it. As for the Keynote session, can you share with us ideas you have for speakers you'd like us to reach out to?

**Comment by jstringh** [comment]
WeCan @ ESRI seem to have a good thing going on. I’m curious to know if they could have a stronger presence at future conferences. Also, considering that they're a career networking group, would they be a good resource for finding a keynote speaker?

**Comment by alexatodd** [comment]
Thank you for your interest in making the Dev Summit more inclusive and diverse. I would suggest that Esri help promote talent by providing the opportunity to be seen. There lies much of the issue.

“...women have a tendency to understate their achievements and, therefore, often need more encouragement to volunteer for speaking events....”

The above quote is a highlight from this article (p 18): https://www.gis-professional.com/magazines/gis-professional-april-2018.pdf

To your question about speakers, I’m interested in hearing Laura Dangermond talk. What about other female leaders at Esri?

**Comment by minottkerr** [comment]
I also thought I noted fewer women this year among the plenary speakers than in previous years. For keynote speakers perhaps tap into the Women in GIS group; they had a strong presence at this year's DevSummit.  Or maybe see if the new ESRI Press book "Women and GIS" might over some leads.

**Comment by juli7579** [comment]
Hey all - I'm Julia, one of the leads for WeCan (thanks @jstringh for the shout-out!). We look forward to having a stronger presence at the UC this year and future conferences, with a presence in sessions and meet ups, but also with formal proposals to the conference organizers on how to improve diversity and inclusion, with a focus on women.  We have included nominations for the UC plenary, and next year we will definitely include nominations for the DevSummit keynote speakers!

@trippetoe @minottkerr @jstringh @alexatodd @jgravois If you have any suggestions on speakers (we have a few already), please share them with us on the [Women's Geospatial Forum](https://community.esri.com/thread/231133-female-keynoter-at-dev-summit)!

**Comment by trippetoe** [comment]
@JimBarry :   With Esri's (and the rest of the tech world's) sprint towards an AI future, how about inviting Cathy O’Neil, author of Weapons of Math Destruction?  It might be interesting to hear about ways that AI can be used and misused.  

From a Data Scientist point of view (my personal favorite):
Emily Robinson - she's done lots of work and speaking on Women the Data Science field 
Megan Robertson - Data Scientist at Nike is doing lots of work in R, and teaching classes at DataCamp

**Comment by JimBarry** [comment]
@trippetoe thanks for the suggestions of Cathy, Emily and Megan. I just put them on our search list.

**Comment by JimBarry** [comment]
@jstringh Thanks for the suggestion of inviting WeCan's involvement.

@juli7579 Thanks so much for jumping in! Wondering if we can phone conf about this, kicking around ideas further?  (cc @Kneeson)

**Comment by JimBarry** [comment]
@minottkerr I hadn't thought to count, but going thru our show script just now, there were 6 women and 20 men in the Plenary session (not counting Jack) ;-). Almost a quarter it seems. Probably roughly the same as the profile of attendees, but I'd have to check. The Esri UC in San Diego definitely has a much more equal breakdown, both in attendees and in Esri staff overall. 

When it comes to our primary criteria for Plenary presenters at DevSummit, so far it has been to choose the most relevant engineer to demonstrate each technology being shown. This engineer often happens to be female, but more often happens to be male. We haven't yet considered gender as a criteria in this selection process, but we can improve by being actively more sensitive to this. We have a large number of female engineers on staff who build ArcGIS. But even when the most qualified engineer in a certain area of the technology is male, doesn't mean that there aren't other female engineers who are fully qualified to do so as well. 

Thanks so much for sharing your thoughts with us on this, and also about involving the Women in GIS group's input. I think it's safe to say that the female-to-male gap is a lot smaller in the GIS user community than it is in the GIS developer community. To encourage and foster change in the latter is a challenge that can be improved on several fronts. Input from you and others who know is greatly appreciated.

**Comment by JimBarry** [comment]
@alexatodd The gender profile at the big Esri UC in San Diego is a lot more equally balanced than at DevSummit, both in attendees, as well as Esri staff on hand, not to mention a lot more involvement of female techs, development leaders, and executives in the Plenary and Closing sessions. It's possible that's a reflection of the gender balance in the GIS user community versus a bit of a different balance in the software/app developer community overall. But as I mentioned in my comment to @minottkerr above, we appreciate suggestions and we are actively interested in improvement and progress. There is surely more we can do. And as all of us involved in GIS know, the proof is and will be in the data. ;-)   As for Laura Dangermond speaking, I'll pass that idea to her. Thanks again for taking the time to share your ideas and suggestions. Very helpful stuff.

**Comment by Dawnabelle** [comment]
I'd also like to point out welcoming non-binary identifying people and more people of color to the plenary stage would be amazing.
